### ISSUE TITLE
Still needs manual compiling every time

### ISSUE THREAD

**Issue by neil-s** [gf3/haml.mode#2]
So if I understand it right, this basically just adds HAML syntax highlighting to Coda? But if I'm using HAML for standalone html development, not Ruby on Rails, I need to compile the haml file to html every time I want to preview it, manually, from the command line? That takes away all of the time advantages of using HAML and Coda. Is there some feature I'm overlooking? Is there a coda plugin to automatically compile the file? If not, I'm going to try to write one, although I've never written Coda extensions before...

**Comment by gf3** [comment]
Yes, you are correct. There is no HAML Coda plugin that I know of. To automatically regenerate your files as you work, try something like this: https://github.com/carlo/haml-sass-file-watcher

**Comment by neil-s** [comment]
That solution still would need me to preview manually in Coda or Espresso. Sorry if I sound picky, but development gets a nice sorta muscle memory rhythm when you can 'look at the code, now look at the live preview, now look at the code again, now back at the preview'. Frustrated at not finding anything, I've begun writing my own plugin for Coda using their dead simple plugin creator, but I'm running into stumbling blocks already. Maybe I'll put up the code and try to get help from others online.

**Comment by gf3** [comment]
You can do that, your editor is Coda and your live preview is your browser (pointed to the generated files).

**Comment by neil-s** [comment]
But the browser pointed to the generated files isn't a live preview. Although yeah, I did try this solution with an autorefresh extension. But that countdown timer made me real uneasy for some reason. Like I had to make some changes quick and switch to the browser window before the time ran out, to make sure it didn't refresh for nothing, and I wouldn't have to wait another 10 seconds for an update. Wow. I'm so messed up. :/
